dbms/src/Storages/Page/V2/PageFile.cpp

```diff
@@ -488,7 +492,7 @@ PageFile::MetaMergingReaderPtr PageFile::MetaMergingReader::createFrom(PageFile
 // Try to initiallize access to meta, read the whole metadata to memory.
 // Status -> Finished if metadata size is zero.
 //        -> Opened if metadata successfully load from disk.
-void PageFile::MetaMergingReader::initialize(std::optional<size_t> max_meta_offset, const ReadLimiterPtr & read_limiter)
+void PageFile::MetaMergingReader::initialize(std::optional<size_t> max_meta_offset, const ReadLimiterPtr & read_limiter, const bool background = false)
```

Comment:
```suggestion
void PageFile::MetaMergingReader::initialize(std::optional<size_t> max_meta_offset, const ReadLimiterPtr & read_limiter, bool background = false)
```

Reply:
why remove `const` here?

Reply:
`const` on arguments that is passed by value is unuseful. But is't not a big deal.